include/crypto/ml_dsa.h

```diff
@@ -110,17 +113,19 @@ __owur int ossl_ml_dsa_pk_decode(ML_DSA_KEY *key, const uint8_t *in, size_t in_l
 __owur int ossl_ml_dsa_sk_decode(ML_DSA_KEY *key, const uint8_t *in, size_t in_len);
 
 EVP_MD_CTX *ossl_ml_dsa_mu_init(const ML_DSA_KEY *key, int encode,
-                                const uint8_t *ctx, size_t ctx_len);
+                                const uint8_t *ctx, size_t ctx_len, int hash);
```

Comment:
```suggestion
                                const uint8_t *ctx, size_t ctx_len, int prehash);
```
Declaration with explicit parameter names may as well matches the definition, and this is a bit more clear.